Resync web platform tests for custom elements

rniwa opened the pull request:
#### 2b0a228021b3362f2a1d6b019ecf1eef10539d0e
<pre>
Resync web platform tests for custom elements
<a href="https://bugs.webkit.org/show_bug.cgi?id=282359">https://bugs.webkit.org/show_bug.cgi?id=282359</a>

Reviewed by Tim Nguyen.

Updated web platform tests for custom elements as of 5543aec78a78289148b99446716400b1cc66640c.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-getName-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-getName.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-builtins-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-bulitins-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-builtins.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-bulitins.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/connected-callbacks-template-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/connected-callbacks-template.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/custom-element-registry/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/custom-element-registry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/element-internals-aria-element-reflection-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/element-internals-aria-element-reflection.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-expected-mismatch.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-ref.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-notref.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-setFormValue-nullish-value.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-target-element-is-held-strongly.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/disabled-delegatesFocus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/disabled-delegatesFocus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-associated-callback.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-disabled-callback-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-disabled-callback.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/label-delegatesFocus-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/label-delegatesFocus.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-disabled.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-disabled.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-readonly.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-readonly.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-disabled.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-disabled.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-readonly.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-readonly.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form-fieldset-elements.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form-fieldset-elements.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-move-inside-form.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-move-inside-form.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-set-form-attr.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-set-form-attr.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-setValidity.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-setValidity.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-datalist.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-datalist.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-disabled-fieldset.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-disabled-fieldset.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-label.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-label.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/htmlconstructor/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/htmlconstructor/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/overwritten-customElements-global-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/overwritten-customElements-global.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/parser/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/parser/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/pseudo-class-defined-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/range-and-constructors.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/Animation.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/Document.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/HTMLElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/HTMLElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLMediaElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLModElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLModElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLOptGroupElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLOptGroupElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLSourceElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/resources/reactions.js:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/resources/garbage-collect.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-createElement.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-createElement.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-innerHTML-upgrade.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-innerHTML-upgrade.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-call.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-call.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-reentry-with-different-definition.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-reentry-with-different-definition.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-criteria.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-criteria.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-order.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-order.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-registry-define-get-etc.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-registry-define-get-etc.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/ElementInternals-states-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/ElementInternals-states-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/ElementInternals-states.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/ElementInternals-states.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/README.md: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/custom-state-set-strong-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-nth-of-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-nth-of.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-pseudo-class-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/state-pseudo-class-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-pseudo-class.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/state-pseudo-class.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/README.md: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/throw-on-dynamic-markup-insertion-counter-construct-xml-parser.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/throw-on-dynamic-markup-insertion-counter-reactions-xml-parser.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/upgrade-custom-element-error-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/upgrade-custom-element-error-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/285944@main">https://commits.webkit.org/285944@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1e5d0a5052ebdb3120a41c74a9eb01ca28d9e381

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/74269 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/53698 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/27080 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/78648 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/25507 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/62831 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/1483 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/58381 "Found 1 new test failure: fast/mediastream/video-mediastream-restricted-invisible-autoplay-user-click.html (failure)") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/16713 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/77336 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/48543 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/63888 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/38793 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/45488 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/21383 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/23840 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/66931 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/21730 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/80165 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/1586 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/901 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/66674 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/1731 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/63906 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/65950 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/9893 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/8047 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/11465 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/1550 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/1579 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/1567 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/1586 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Can you add the upstream commit to the commit message?

rniwa committed:
Resync web platform tests for custom elements
https://bugs.webkit.org/show_bug.cgi?id=282359

Reviewed by Tim Nguyen.

Updated web platform tests for custom elements as of 5543aec78a78289148b99446716400b1cc66640c.

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-getName-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry-getName.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/CustomElementRegistry.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-accessibility.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-builtins-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-bulitins-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-builtins.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-customized-bulitins.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/HTMLElement-constructor.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/connected-callbacks-template-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/connected-callbacks-template.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/custom-element-registry/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/custom-element-registry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/element-internals-aria-element-reflection-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/element-internals-aria-element-reflection.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-expected-mismatch.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-ref.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-notref.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-reportValidity-bubble.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-setFormValue-nullish-value.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/ElementInternals-target-element-is-held-strongly.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/disabled-delegatesFocus-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/disabled-delegatesFocus.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-associated-callback.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-disabled-callback-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/form-disabled-callback.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/label-delegatesFocus-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/label-delegatesFocus.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-disabled.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-disabled.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-readonly.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-constructor-readonly.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-disabled.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-disabled.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-readonly.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/attr-parser-readonly.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form-fieldset-elements.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form-fieldset-elements.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-form.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-move-inside-form.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-move-inside-form.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-set-form-attr.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-set-form-attr.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-setValidity.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/constructor-setValidity.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-datalist.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-datalist.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-disabled-fieldset.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-disabled-fieldset.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-label.tentative-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/upgrading/inside-label.tentative.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/form-associated/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/htmlconstructor/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/htmlconstructor/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/overwritten-customElements-global-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/overwritten-customElements-global.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/parser/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/parser/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/pseudo-class-defined-print-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/range-and-constructors.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/Animation.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/AriaMixin-string-attributes.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/Document.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/HTMLElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/HTMLElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLMediaElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLModElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLModElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLOptGroupElement-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLOptGroupElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/HTMLSourceElement.html:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/customized-builtins/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/resources/reactions.js:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/reactions/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/resources/garbage-collect.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-createElement.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-createElement.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-innerHTML-upgrade.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/ShadowRoot-innerHTML-upgrade.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-call.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-call.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-reentry-with-different-definition.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/constructor-reentry-with-different-definition.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-criteria.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-criteria.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-order.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-define-upgrade-order.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-registry-define-get-etc.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/scoped-registry-registry-define-get-etc.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/scoped-registry/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/ElementInternals-states-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/ElementInternals-states-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/ElementInternals-states.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/ElementInternals-states.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/README.md: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/custom-state-set-strong-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-nth-of-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-nth-of.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-css-selector.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-pseudo-class-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/state-pseudo-class-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/state-pseudo-class.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/state-pseudo-class.html.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/README.md: Removed.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/custom-elements/state/tentative/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/throw-on-dynamic-markup-insertion-counter-construct-xml-parser.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/throw-on-dynamic-markup-insertion-counter-reactions-xml-parser.xhtml:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/upgrade-custom-element-error-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/upgrade-custom-element-error-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/upgrading/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/custom-elements/w3c-import.log:
* LayoutTests/platform/gtk/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/wpe/TestExpectations:
* LayoutTests/platform/wpe/imported/w3c/web-platform-tests/custom-elements/ElementInternals-role-expected.txt: Added.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/285944@main
